x-pack/packages/kbn-elastic-assistant/impl/data_anonymization_editor/get_stats/index.ts

```diff
@@ -13,9 +13,11 @@ import { Stats } from '../helpers';
 export const getStats = ({
   anonymizationFields = [],
   rawData,
+  replacements,
 }: {
   anonymizationFields?: AnonymizationFieldResponse[];
   rawData?: string | Record<string, string[]>;
+  replacements?: Record<string, string>;
```

Comment:
Nit, we have a type in common attributes we could use for this and other instances of replacements
```suggestion
  replacements?: Replacements;
```